_includes/socials.html

```diff
@@ -17,7 +17,7 @@
     <div class="width-3-12 width-0-12-m"></div>
     <div class="width-3-12 width-0-12-m"></div>
     <div class="width-6-12 width-12-12-m">
-      <a href="http://www.twitter.com/@quarkusio" class="button-cta"><i class="fab fa-twitter"></i>Follow us on Twitter</a>
+      <a href="https://x.com/@quarkusio" class="button-cta"><i class="fab fa-twitter"></i>Follow us on X</a>
```

Comment:
Unfortunately, FA 5 doesn't support X so we would have to move to FA 6.
Not sure the update will be straightforward and we need to make sure we can still use it for free.